<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="temprature">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>841</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Gestion de client</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">tabWidget_2 = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100)</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ajout</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>nom_cli</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_cli</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>prenom_cli</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>nombre_com</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prenom">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nombre">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Ajouter">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nom">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>10</y>
          <width>471</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>340</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_cli</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_idsupprimer">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>340</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer_3">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="push_recherche_3">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>380</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>recherche</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_recherche_3">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>380</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_44">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_cli</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>340</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>prenom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>390</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>430</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nbr</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="idclient">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>300</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="nomclient">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>340</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="prenomclient">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>390</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="nombreclient">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>430</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>430</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="trie">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>420</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id_cli</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nom_cli</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>prenom_cli</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nombre_com</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>420</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TRIE</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="push_recherche">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>610</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_recherche_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>610</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion des commande</string>
     </attribute>
     <widget class="QTabWidget" name="affichage2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>771</width>
        <height>601</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Ajout</string>
       </attribute>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>date_com</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nombre_come</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_com">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_nb">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_42">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_com</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_date">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>0</y>
          <width>471</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_idsupprimer_2">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>340</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">m_myWidget = new QWidget(this); m_myWidget-&gt;setGeometry(0, 0, 300, 100); QPalette pal = palette();</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_43">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>340</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>id_com</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QComboBox" name="triecommande">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>380</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>id_com</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>date_com</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nombre_come</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="linecomande">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>410</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>380</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TRIE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>410</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>rechercher</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>240</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>270</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>300</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nombre</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="idcom">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>240</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="datecome">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>270</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="nbcom">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>300</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="modifiercom">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>330</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="supprimer_4">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>590</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="trie_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>590</y>
        <width>73</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id_cli</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom_cli</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prenom_cli</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nombre_com</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>600</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TRIE</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
